packages/messaging/android/src/main/java/io/invertase/firebase/messaging/ReactNativeFirebaseMessagingModule.java

```diff
@@ -120,8 +121,9 @@ public void setAutoInitEnabled(Boolean enabled, Promise promise) {
   }
 
   @ReactMethod
-  public void getToken(Promise promise) {
-    Tasks.call(getExecutor(), () -> Tasks.await(FirebaseMessaging.getInstance().getToken()))
+  public void getToken(String projectId, String senderId, Promise promise) {
+    FirebaseMessaging messagingInstance = FirebaseApp.getInstance(projectId).get(FirebaseMessaging.class);
```

Comment:
```suggestion
  public void getToken(String appName, String senderId, Promise promise) {
    FirebaseMessaging messagingInstance = FirebaseApp.getInstance(appName).get(FirebaseMessaging.class);
```

Reply:
Interesting, I think after I approve it I can no longer do the "make comment with suggestion" / "commit using github UI" thing. No worries, will pull local and patch up, only takes a moment